src/_rules/slider.js

```diff
@@ -14,3 +14,13 @@ export const sliderHandleShadow = [
     }),
   ],
 ];
+
+export const sliderOutlineNone = [
+  [
+    /^slider-handle-outline-none$/,
+    () => ({
+      outline: 'rgba(0, 0, 0, 0) solid 2px',
+      'outline-offset': '2px',
```

Comment:
I remember that there was an `outline-none` class before that would apply the `outline` and `outline-offset` as you did here, but it was said to be deprecated. Is this a use-case where we'd consider bringing it back? @Skadefryd @adidick @AnnaRybkina?

Reply:
We do have https://github.com/warp-ds/drive/blob/alpha/src/_rules/focus-ring.js#L13 that adds `outline-none` but it is totally different I guess.

Reply:
I would suggest to add a very clear TODO here(if we are going with this solution) that this way is not a preferred way of adding styling.
@Skadefryd is also suggesting that we add `outline-offset` rule to drive to handle line 23 as it is done in [TW](https://tailwindcss.com/docs/outline-offset)

Reply:
Okay agree, but we can go with this solution for now and add a todo, as it's a copy from the fabric solution? And we can modify it later when outline-offset is in place?